nautobot/extras/test_jobs/singleton.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+from nautobot.core.celery import register_jobs
+from nautobot.extras.jobs import Job
+
+
+class TestSingletonJob(Job):
+    class Meta:
+        name = "Example Singleton Job"
+        is_singleton = True
+
+    def run(self):
+        time.sleep(60)
```

Comment:
Maybe reduce this number somewhat, or make it configurable as a job input, so that running the added tests don't take a minute each?